### ISSUE TITLE
Button for archiving cells

### ISSUE THREAD

**Issue by jmadden173** [jlab-sensing/ENTS-backend#111]
Once a deployment is done (as is the case with Bill's deployment), we are not actively monitoring the cell data but still want to retain the data. An "archive" button that removes the cells from the list would help to only show relevant data which would prevent confusion.

**Comment by jmadden173** [comment]
Duplicate of #225
